esmf_regrid/experimental/unstructured_scheme.py

```diff
@@ -476,7 +539,7 @@ def _regrid_along_grid_dims(regridder, data, grid_x_dim, grid_y_dim, mdtol):
     return result
 
 
-def _create_mesh_cube(data, src_cube, grid_x_dim, grid_y_dim, mesh):
+def _create_mesh_cube(data, src_cube, grid_x_dim, grid_y_dim, mesh, location="face"):
```

Comment:
```suggestion
def _create_mesh_cube(data, src_cube, grid_x_dim, grid_y_dim, mesh, location):
```

I've been through the call stack. This is always going to receive a `location`.

(Indeed if it doesn't receive a `location` that's probably breaking some assumptions about how the code is working and we want an `Error` rather than silent continuation).